Update uninstall (#1859)
hashicorp/terraform-website: content/source/docs/enterprise/install/uninstall.html.md

1. Enter `yes` when asked if you want to continue with the installation.

	```

> 	$ sudo ./uninstall.sh 

Reviewer: ```suggestion
	$ sudo ./uninstall.sh
```

	This script will completely uninstall Terraform Enterprise and Replicated on this system, as well as remove associated files.
	Do you wish to continue? (y/n)yes
	Proceeding with uninstall...
	```

	If there are snapshots present on the system, you will be prompted to move them, proceed with deleting them, or abort.
	
	```
	There appear to be Replicated snapshots stored in /var/lib/replicated/snapshots.
	1) Move the snapshots to another directory
	2) Continue uninstall and delete the snapshots
	3) Cancel the uninstall
	Select an option: 1

> 	Enter the directory to move the snapshots to:/tmp

Reviewer: ```suggestion
	Enter the directory to move the snapshots to: /tmp
```

Author: I knew someone was gonna comment on that. I gotta update the script to fix this. (This is all copy-pasta directly from actual script output.

	The snapshots will be moved to /tmp.

> 	Press Y to continue or N to cancel.yMoving snapshots...

Reviewer: ```suggestion
	Press Y to continue or N to cancel. y
	Moving snapshots...
```

	```
	$ sudo ./uninstall.sh
	This script will completely uninstall Terraform Enterprise and Replicated on this system, as well as remove associated files.
	Do you wish to continue? (y/n)yes
	Proceeding with uninstall...
	```

> 	If there are snapshots present on the system, you will be prompted to move them, proceed with deleting them, or abort.

Reviewer 2: ```suggestion
	If there are snapshots present on the system, choose whether to move them to another directory, delete them, or cancel the uninstall. Here is an example of moving snapshots to another directory:
```

	
	```
	There appear to be Replicated snapshots stored in /var/lib/replicated/snapshots.
	1) Move the snapshots to another directory
	2) Continue uninstall and delete the snapshots
	3) Cancel the uninstall
	Select an option: 1
	Enter the directory to move the snapshots to: /tmp
	The snapshots will be moved to /tmp.
	Press Y to continue or N to cancel. y
	Moving snapshots...
	Files moved.
	```

> 	The uninstall will then continue...

Reviewer 2: I would do this here, and remove that line from below the example block. I think this is clearer. 

```suggestion
	The script continues and stops the Replicated services, removes the Docker containers, and removes Replicated executables and configuration files from the system.  
```
